src/server/system_services/stats_aggregator.js

```diff
@@ -507,9 +507,11 @@ async function _partial_buckets_info(req) {
             const bucket_available = size_utils.json_to_bigint(_.get(bucket_info, 'data.free') || 0);
             const bucket_total = bucket_used.plus(bucket_available);
             const is_capacity_relevant = _.includes(CAPACITY_MODES, bucket_info.mode);
+            const { size_used_percent, quantity_used_percent } = Quota.get_bucket_quota_usages_percent(bucket, new Quota(bucket.quota));
```

Comment:
Why use `Quota` and not call its contractor first?

Reply:
The methods were static, but I already moved them to the class. Thanks